src/world.rs

```diff
@@ -121,6 +128,65 @@ impl World {
     }
 }
 
+impl World {
+    /// Move the components of an [Entity] from one world to another
```

Comment:
Maybe make it a bit more clear by replacing `from one world` by `from this world`?